activity/covid19/covid19_schema

```diff
@@ -1,100 +1,103 @@
 {
     "@context": [
-        "https://raw.githubusercontent.com/ReproNim/reproschema/master/contexts/generic",
-        "https://raw.githubusercontent.com/sensein/covid19/master/activity/covid19/covid19_context"
+        "https://raw.githubusercontent.com/ReproNim/reproschema/refactor-1.0/contexts/generic",
+        {
+            "wellbeing": "https://raw.githubusercontent.com/ReproNim/reproschema/master/activities/Wellbeing/items/",
+            "nda": "https://raw.githubusercontent.com/ReproNim/reproschema/master/activities/NDA/items/"
```

Comment:
@danielmlow - nothing should reference the library in master anymore.

Reply:
Sorry, thought I was on my branch. Didn't mean to do a PR. I'd be happy to fix it, but don't know all the details. What should it be referencing instead?

Reply:
reproschema-library